### DISCUSSION TITLE
unintended scientific notation in table field

### DISCUSSION THREAD

**Discussion by fantismic** [rappasoft/laravel-livewire-tables#2001] [Q&A] [answered]
I have search and notice there's already a bug reported but regarding the URL.
I dont know if this is the same thing or not, but im pulling some data using sushi from an api that have a field with ticket ID, these ids are 8 chars lenght but sometimes they are only numbers (with an E) like "2488E260" that are being display in the table as "2.488E+263"
The main problem here is that users search by this ID, so i need it to be right.
Is there a workaround or fix that doesnt break the search while displaying the data correctly?

Thanks in advance for your help.

**Comment by lrljoe** [comment]
So this is actually a bug with Livewire Core, rather than the tables package itself sadly.  I'll chase up the Livewire Core team and see if there's any timescale for a resolution.

**Comment by lrljoe** [comment, marked as answer]
This should be fixed in the next core Livewire release I believe
